app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :likes
+  has_many :posts
+  has_many :comments
+end
```

Comment:
- [ ] It would be better if the User model has a method that returns `3 most recent posts`